packages/ui/src/typography.styles.css

```diff
@@ -51,7 +51,7 @@ body {
 }
 
 .label-large {
-    @apply text-sm font-normal leading-[160%] text-gray-800 sm:text-base sm:font-medium sm:leading-[124%];
+    @apply text-sm font-normal leading-[160%] text-gray-800 sm:text-base sm:font-medium sm:leading-[20px];
```

Comment:
I guess it's better to still use percent values for better scalability, here it would be 125%

Reply:
https://github.com/prezly/theme-kit-js/pull/404/commits/c3ea8f70811b98b5dc627d4a305b2537ababbcbc